moduleroot/Rakefile.erb

```diff
@@ -16,3 +16,7 @@ import '<%= item %>'
 <% checks.each do |check| -%>
 PuppetLint.configuration.send('<%= check %>')
 <% end -%>
+
+<%- [@configs['extras']].flatten.compact.each do |line| -%>
```

Comment:
This will fail if there are no `extras` configured for a module. Either add a default empty array to the config_defaults, or check here for `nil`-ness.

Reply:
@DavidS very good point. Sorry about that. The default array have been added.